stl/inc/xtr1common

```diff
@@ -175,13 +175,17 @@ _EXPORT_STD _NODISCARD constexpr bool is_constant_evaluated() noexcept {
 }
 #endif // _HAS_CXX20
 
+struct _Signed128;
+struct _Unsigned128;
+
 _EXPORT_STD template <class _Ty>
-_INLINE_VAR constexpr bool is_integral_v = _Is_any_of_v<remove_cv_t<_Ty>, bool, char, signed char, unsigned char,
-    wchar_t,
+_INLINE_VAR constexpr bool is_integral_v =
+    _Is_any_of_v<remove_cv_t<_Ty>, bool, char, signed char, unsigned char, wchar_t,
 #ifdef __cpp_char8_t
-    char8_t,
+        char8_t,
 #endif // __cpp_char8_t
-    char16_t, char32_t, short, unsigned short, int, unsigned int, long, unsigned long, long long, unsigned long long>;
+        char16_t, char32_t, short, unsigned short, int, unsigned int, long, unsigned long, long long,
+        unsigned long long, _Signed128, _Unsigned128>;

```

Comment:
It's non-conforming to add `_Signed128` and `_Unsigned128` here because they are class types. Is there any reason for this?

Reply:
I think making int128 true for is_integral & is_arithmetic would make it easier for people writing generic arithmetic functions.

Reply:
remove in [`acf5ff6` (#3559)](https://github.com/microsoft/STL/pull/3559/commits/acf5ff67287eda06a5ed78d013ec3254ec3aa48b)
thanks